skyve-war/src/main/java/modules/admin/UserLoginRecord/UserLoginRecord.xml

```diff
@@ -20,5 +20,13 @@
 		<boolean name="failed" required="true" usage="domain">
 			<displayName>admin.userLoginRecord.failed.displayName</displayName>
 		</boolean>
+		<text name="ipAddress" required="true" usage="domain" sensitivity="personal">
+			<displayName>admin.userLoginRecord.ipAddress.displayName</displayName>
+			<length>254</length>
+		</text>
+		<text name="countryCode" usage="domain">
+			<displayName>admin.userLoginRecord.countryCode.displayName</displayName>
```

Comment:
Please add a `documentation` to what this is used for, and when it is not used. i.e. it is only used if enabled.